docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,195 @@
+===================
+Trino on Kubernetes
+===================
+
+`Kubernetes <https://kubernetes.io>`_ is a container orchestration platform that
+allows you to deploy Trino and other applications in a repeatable manner across
+different types of infrastructure. This can range from deploying on your laptop
+using tools like `KIND <https://kind.sigs.k8s.io>`_ and
+`Helm <https://helm.sh>`_, to running on a managed Kubernetes service on the
+major clouds like `Amazon EKS <https://aws.amazon.com/eks>`_,
+`Google GKE <https://cloud.google.com/kubernetes-engine>`_, and
+`Azure AKS <https://azure.microsoft.com/en-us/services/kubernetes-service>`_.
```

Comment:
GKE means Google Kubernetes Engine, so saying Google GKE seems wrong. Same for Azure Kubernetes Service. Maybe don't use abbreviations.

Reply:
Agreed...lets use the long form names for all of them  Amazon Elastics Kubernetes Service (EKS) 

see https://docs.starburst.io/starburst-enterprise/k8s/index.html for list